source/algorithms.tex

```diff
@@ -4057,7 +4057,7 @@
 \pnum
 \returns \tcode{true} if
 \range{first}{last} is empty or if
-\range{first}{last} is partitioned by \tcode{pred}, i.e. if all elements that satisfy \tcode{pred} appear before those that do not.
+\range{first}{last} is partitioned by \tcode{pred}, i.e., if all elements that satisfy \tcode{pred} appear before those that do not.
```

Comment:
More damage to the English language perpretrated by Strunk and White, but OK, this is the dominant American English style, so let's put up with it.